src/main/java/com/julienviet/pgclient/impl/RowImpl.java

```diff
@@ -131,4 +132,9 @@ public OffsetDateTime getOffsetDateTime(String name) {
   public UUID getUUID(String name) {
     return getUUID(desc.columnIndex(name));
   }
+
+  @Override
+  public BigDecimal getBigDecimal(String name) {
+    return getBigDecimal(desc.columnIndex(name));
```

Comment:
we need to handle the case where columnIndex returns -1

Reply:
@vietj is this should be handled for `BigDecimal` only ?

Reply:
no, it should always be handled, perhaps we should handle it in a separate patch

Reply:
ok

Reply:
it seems actually is should be fine, we should check at least these cases are tested.

Reply:
https://github.com/vietj/reactive-pg-client/issues/68